templates/documents/general_upload.html

```diff
@@ -1,79 +0,0 @@
-
-<form method=post enctype="multipart/form-data" action="{$FORM_ACTION}" onsubmit="return top.restoreSession()">
```

Comment:
this templates/documents/general_upload.html script is feeling unwanted in your PRs :)